README.md

```diff
@@ -248,6 +248,39 @@ module.exports = {
 }
 ```
 
+### Locales
+
+Often we render multiple versions of our application for different locations. Eg. Australia & New Zealand. To render an html file for each location you can use the locales option in `sku.config.js`.
+
+Locales accepts an array of strings representing each locale you want to render html files for.
+
+```js
+module.exports = {
+  locales: ['AU', 'NZ']
+}
+```
+
+For each locale, sku will call your `render.js` function and pass it the locale as a local.
```

Comment:
Feel like using "locale" and "local" so close together is confusing. In fact, in this case I think you could avoid using the word "local" entirely.